resources/js/Shared/ContactCard.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div class="relative inline" :class="divOuterClass" :style="'top: ' + top">
```

Comment:
This `divOuterClass` is useless here. Simply define a `class` attribute on the caller and those classes will be add in the root node (if needed, but you did not use it apparently).

Reply:
fixed